### DISCUSSION TITLE
Database support

### DISCUSSION THREAD

**Discussion by FairInHeight** [EssentialsX/Essentials#4306] [Ideas and Feature Suggestions] [answered]
Honestly this is something so mainstreamed and useful that I just assumed you guys supported MySQL. This is something that is not only useful, but crucial for those of us running bungee networks. I love essentials, but due to it's lack of bungee/database support I will now have to use other plugins.

**Comment by pop4959** [comment, marked as answer]
EssentialsX has no plans to add database support (nor is it really necessary for the overwhelming majority of the plugin, if at all).
